src/crossfilter.js

```diff
@@ -68,6 +69,22 @@ function crossfilter() {
     triggerOnChange('dataRemoved');
   }
 
+  // Return true if the data element at index i is filtered IN.
+  // Optionally, ignore the filters of any dimensions in the ignore_dimensions list.
+  function isElementFiltered(i, ignore_dimensions) {
+      var n,
```

Comment:
I think we are generally using 2-space indentation. If you can switch to that for this change, it would be helpful. (In this file and in the test.)